.github/workflows/codeql.yml

```diff
@@ -0,0 +1,91 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL Advanced"
+
+on:
+    push:
+        branches: [ "main", "rc-*" ]
+    pull_request:
+        branches: [ "main", "rc-*" ]
+    schedule:
+        - cron: '25 4 * * 5'
```

Comment:
Better on monday? So we could fix it quickly instead of having it broken during the weekend

Reply:
this was github's generated default, when converting from the basic CodeQL scanning (default) to the advanced (explicit yaml-defined workflow you can customize)